src/docs/environment.mdx

```diff
@@ -164,3 +164,122 @@ If you would like to be able to run `devserver` outside of your root checkout, y
+
+**Solution:** Your virtualenv needs to be updated. In sentry, type `make install-sentry-dev`. In getsentry, type `make install-develop-requirements`.
+
+---
+
+**Problem:** You see `Error occured while trying to proxy to: dev.getsentry.net:8000/`
+
+**Solution:**  You likely need to upgrade your Python dependencies. Go to the git root directory and run `make install-sentry-dev`.
+
+---
+
+**Problem:**  "out of memory" issues after running the sentry or getsentry devserver, or during webpacking.
+
+**Solution:**
+
+```bash
+export NODE_OPTIONS="--max-old-space-size=4096"
+```
+
+---
+
+**Problem:** `Module not found: Error: Can't resolve 'integration-docs-platforms'`
+
+**Solution:**
+
+```bash
+make build-platform-assets
+```
+
+---
+
+**Problem:** `SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 76`
+
+Also, another symptom indicating you need to upgrade chromedriver:
+
+```shell
+Traceback (most recent call last):
+  File "/Users/joshua.li/dev/sentry/sentry/src/sentry/utils/pytest/selenium.py", line 344, in browser
+    driver = start_chrome(**chrome_args)
+  File "/Users/joshua.li/dev/sentry/sentry/src/sentry/utils/retries.py", line 41, in execute_with_retry
+    return retrier(functools.partial(fn, *args, **kwargs))
+  File "/Users/joshua.li/dev/sentry/sentry/src/sentry/utils/retries.py", line 85, in __call__
+    error,
+RetryException: Could not successfully execute <functools.partial object at 0x10f31e7e0> within 15.830 seconds (12 attempts.)
+```
+
+**Solution:**
+
+```bash
+brew cask upgrade chromedriver
+```
+
+---
+
+**Problem:**
+
+```shell
+Traceback (most recent call last):
+  ...
+django.core.exceptions.ImproperlyConfigured:
+Application labels aren't unique, duplicates: okta
+make: *** [develop] Error 1
+```
+
+**Solution:**
+
+```jsx
+pip uninstall sentry-auth-github sentry-auth-saml2
+```
+
+---
+
+**Problem:**
+
+```shell
+00:51:27 server  | Python auto-reloader enabled
+00:51:27 server  |   File "/Users/dfuller/code/sentry/src/sentry/wsgi.py", line 11, in <module>
+00:51:27 server  |     from django.conf import settings
+00:51:27 server  |   File "/Users/dfuller/code/sentry/.venv/lib/python2.7/site-packages/django/conf/__init__.py", line 14, in <module>
+00:51:27 server  |     from django.core.exceptions import ImproperlyConfigured
+00:51:27 server  |   File "/Users/dfuller/code/sentry/.venv/lib/python2.7/site-packages/django/core/exceptions.py", line 5, in <module>
+00:51:27 server  |     from django.utils.encoding import force_text
+00:51:27 server  |   File "/Users/dfuller/code/sentry/.venv/lib/python2.7/site-packages/django/utils/encoding.py", line 10, in <module>
+00:51:27 server  |     from django.utils.functional import Promise
+00:51:27 server  |   File "/Users/dfuller/code/sentry/.venv/lib/python2.7/site-packages/django/utils/functional.py", line 1, in <module>
```

Comment:
@markstory it might be good to `-- snip --` the lines above here, in the interest of leaner docs.